.github/workflows/merge-main.yml

```diff
@@ -120,7 +121,9 @@ jobs:
           - name: api
             file: api/openshift.deploy.yml
             overwrite: true
-            parameters: -p URL=fom.nrs.gov.bc.ca
```

Comment:
line 109 looks wrong - should be name: PROD Deploys I believe. Likely a Derek copy/paste error

Reply:
Changed.